bundles/core/src/main/java/com/adobe/cq/commerce/demandware/preview/StaticContentProxyServlet.java

```diff
@@ -53,17 +53,25 @@
 
     @Reference
     private DemandwareClientProvider clientProvider;
-
+    
+    @Reference
+    private InstanceIdProvider instanceId;
+    
     @Override
     protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse response) throws ServletException,
             IOException {
 
         RequestPathInfo pathInfo = request.getRequestPathInfo();
+        Optional<DemandwareClient> demandwareClient = clientProvider.getClientForSpecificInstance(instanceId.getInstanceId(request));
+        if(!demandwareClient.isPresent()){
+            LOG.error("FailedtogetDemandwareClientfor[{}]DemandwareinstanceId.",instanceId.getInstanceId(request));
```

Comment:
error already logged in getClientForSpecificInstance, so no need for duplicate message